compatibility/README.md

```diff
@@ -0,0 +1,62 @@
+Unlike the web, which has just a few browsers handling most traffic, emails are read in a huge variety of email applications, or clients. 
+
+The varying levels of support across email clients and their rendering engines leads to a lot of display issues, which is why the emails need to constantly be tested before pushing send. The code generated by our email HTML generator, that is used by the EDMdesigner and Chamaileon editor, is tested frequently on the major email clients and email preview services as Email On Acid and Litmus.
+
+Supported email clients:
+
+###Desktop clients:
+
+Air Mail 2.0
+Apple Mail 4, 5, 6, 7, 8, 9
+Entourage 2004, Entourage 2008 - dicontinued
+Microsoft Outlook for Windows 2007, 2010, 2011, 2013, 2016, 2019 *
+Thunderbird 3.0+
+Windows Live Mail -discontinued, but usable with non-Microsoft email accounts
+Windows Mail
+
+###Web based:
+
+AOL Mail
+Atmail Open
+Cit<span>roMail.h</span>u
+Comcast
+D<span>aum.n</span>et
+Fr<span>eeMail.h</span>u
+Google Inbox - was shut down on 2019
+Gmail
+GMX
+G Suite
+Li<span>bero.i</span>t
```

Comment:
Mik ezek a spanok? :)

Reply:
az azert kell, hogy ne autolinkelje be, mert kozben nincs hozza kotve valodi link es megteveszto